themes/ddbasic/sass/module/_ding_nodelist.scss

```diff
@@ -0,0 +1,1101 @@
+/* Colors variables
+   ========================================================================== */
+
+/* Widgets
+   ========================================================================== */
+
+.ding_nodelist-simple,
+.ding_nodelist-simple > div {
+  float: left;
+}
+
+.ding_nodelist-carousel {
+  height: 200px;
+}
+
+/* Layout
+   ========================================================================== */
+
+.ding_nodelist-col-first {
+  clear: left;
+}
+
+.ding_nodelist-col-full {
+  width: 100%;
```

Comment:
These widths seems related to the site grid. If that is the case then please use the functionality provided by zengrids ([example](https://github.com/ding2/ding2/blob/master/modules/ding_frontend/plugins/layouts/single_row/25_50_25/25_50_25.admin.scss)).